modules/sandboxed-containers-selecting-nodes.adoc

```diff
@@ -3,5 +3,68 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-selecting-nodes_{context}"]
+= Selecting nodes for {sandboxed-containers-first}
 
-= Selecting nodes for OpenShift sandboxed containers
+You can selectively install the Kata runtime on specific workers.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
```

Comment:
Same as the above, should it be `cluster-admin` or just `admin` here? @jensfr @zanetworker ?

Reply:
cluster-admin. Thats also what's used across the docs.

Reply:
Real cluster-admin in this case even I believe, because you need to install the operator and I think that creates resources outside of the operator namespace, for example in the openshift-marketplace namespace. Not 100% sure but I think it's safe to use cluster-admin here.